### ISSUE TITLE
Dota Plus Rare Item(Courier)

### ISSUE THREAD

**Issue by Quiel12** [ValveSoftware/Dota2-Gameplay#15824]
### Cosmetic Item Name

Sir Molestach Irondrill

### Description

When I unbundled the courier and loading screen in it, I didn't get a random effect and color gems in my courier. I spent many dota plus shards to get the courier for the gem but my courier doesn't have any right now, it's like plane old courier. The attachment on the screenshot message is a photo of my courier without gems and random effects. 
Please help this is my Steam ID: 76561198404859413.  I don't have a match that played with my courier is equipped.

### Example Match ID

7624332054

### Screenshots

![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162518766/e9c843ec-fb57-4c97-96a0-b6764939ca43)

**Comment by Zenxseven** [comment]
Same goes with mine.

**Comment by thanhdzhp** [comment]
Yeah i got it yesterday and when i opened it nothing happened. No gem, no primastic..

**Comment by thanhdzhp** [comment]
Yeah i got this courier yesterday but when i opened it nothing happened, no gem, no primastic...
![IMG_20240307_085419](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162525939/b3dbc694-03b5-44a4-a6f2-34a2c9844799)
![IMG_20240306_191008](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162525939/f0523ea6-59f7-474d-bf98-746c12c6dfd6)
Steam id : 1565721627

**Comment by commentator24** [comment]
#15770

**Comment by destinyismy** [comment]
![20240306233434_1](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162527655/be84f6f5-918b-4d96-bb54-f203fd32a0a3)

Same Open on 06/03/2024 night. No gem and unusual effects

**Comment by pakzs** [comment]
![Captura de pantalla 2024-03-07 114422](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162613149/ae966f9b-e5fa-411f-9377-1cca4116371a)
There are no gems.
Steam ID: 76561198363598554

**Comment by KochaAm** [comment]
Same Open on 08/03/2024 night.
Steam ID: 1072175583
![image](https://github.com/ValveSoftware/Dota2-Gameplay/assets/162627263/fd84a12d-0285-4e94-b2bc-9a92e20f18f5)
